inc/samltud-helper.php

```diff
@@ -53,4 +53,4 @@ private function getUsername($user) {
 
 }
 
-$samltud_helper = new SAMLTUD_Helper();
+// $samltud_helper = new SAMLTUD_Helper();
```

Comment:
Kunnen deze // nog even terug? Dit kan beter in een losse commit, zodat, als er iets fout gaat, we het makkelijk ongedaan kunnen maken